packages/react-meteor-data/README.md

```diff
@@ -328,13 +332,109 @@ const docs = useFind(() => {
+disposed.)
 
-Ideally, side-effects such as creating a Meteor computation would be done in `useEffect`. However, this is problematic for Meteor, which mixes an initial data query with setting up the computation to watch those data sources all in one initial run. If we wait to do that in `useEffect`, we'll end up rendering a minimum of 2 times (and using hacks for the first one) for every component which uses `useTracker` or `withTracker`, or not running at all in the initial render and still requiring a minimum of 2 renders, and complicating the API.
+Ideally, side-effects such as creating a Meteor computation would be done in `useEffect`. However, this is problematic
+for Meteor, which mixes an initial data query with setting up the computation to watch those data sources all in one
+initial run. If we wait to do that in `useEffect`, we'll end up rendering a minimum of 2 times (and using hacks for the
+first one) for every component which uses `useTracker` or `withTracker`, or not running at all in the initial render and
+still requiring a minimum of 2 renders, and complicating the API.
 
-To work around this and keep things running fast, we are creating the computation in the render method directly, and doing a number of checks later in `useEffect` to make sure we keep that computation fresh and everything up to date, while also making sure to clean things up if we detect the render has been tossed. For the most part, this should all be transparent.
+To work around this and keep things running fast, we are creating the computation in the render method directly, and
+doing a number of checks later in `useEffect` to make sure we keep that computation fresh and everything up to date,
+while also making sure to clean things up if we detect the render has been tossed. For the most part, this should all be
+transparent.
+
+The important thing to understand is that your reactive function can be initially called more than once for a single
+render, because sometimes the work will be tossed. Additionally, `useTracker` will not call your reactive function
+reactively until the render is committed (until `useEffect` runs). If you have a particularly fast changing data source,
+this is worth understanding. With this very short possible suspension, there are checks in place to make sure the
+eventual result is always up to date with the current state of the reactive function. Once the render is "committed",
+and the component mounted, the computation is kept running, and everything will run as expected.
+
+
+## Suspendable version of hooks
+
+### `suspense/useTracker`
+
+This is a version of `useTracker` that can be used with React Suspense. It is similar to `useTracker`, 
+but it throws a promise and suspends the rendering until the promise is resolved. Also in this version it has the need
+for a key as its first parameters. the key is used to identify the computation and to avoid recreating it when the 
+component is re-rendered.
```

Comment:
Overall it's fine, but I just think that maybe we should focus more on _why_ and not _how_ - parts like "it throws a promise" are an implementation detail of the Suspense for data fetching (it's still experimental!). Also, please review it for grammar, e.g., "the key is used [...]" should start with a capital letter.